local-cli/runIOS/runIOS.js

```diff
@@ -25,15 +27,37 @@ function runIOS(argv, config, args) {
   const scheme = args.scheme || inferredSchemeName;
   console.log(`Found Xcode ${xcodeProject.isWorkspace ? 'workspace' : 'project'} ${xcodeProject.name}`);
 
+  if (args.device) {
+    const devices = parseIOSDevicesList(
+      child_process.execFileSync('xcrun', ['instruments', '-s'], {encoding: 'utf8'})
```

Comment:
Hi grabbou, 
yea sadly this only applies to `xcrun simctl list devices --json`,  which only lists simulators. 
I could implement it for the parseIOSSimulatorsList function. However the command i'm using to receive the real devices is `xcrun instruments -s`, which does not have a `json` flag as far as i know.